src/nginx/status.cc

```diff
@@ -317,6 +317,15 @@ ngx_int_t ngx_esp_init_process_stats(ngx_cycle_t *cycle) {
   for (ngx_uint_t i = 0, napis = mc->endpoints.nelts; i < napis; i++) {
     ngx_esp_loc_conf_t *lc = endpoints[i];
     if (lc->esp) {
+      std::string rollouts;
+      ngx_esp_update_rollout(lc->esp, &rollouts);
+      if (rollouts.length() <= kMaxServiceRolloutsInfoSize) {
+        process_stat->esp_stats[process_stat->num_esp].rollouts_length =
+            rollouts.length();
+        strncpy(process_stat->esp_stats[process_stat->num_esp].rollouts,
```

Comment:
can we compare first, if changed, update share memory.  For share memory, less update is better.